src/sonic-config-engine/tests/test_minigraph_case.py

```diff
@@ -548,3 +548,14 @@ def test_parse_device_desc_xml_mgmt_interface(self):
         self.assertEqual(len(mgmt_intf.keys()), 1)
         self.assertTrue(('eth0', 'FC00:1::32/64') in mgmt_intf.keys())
         self.assertTrue(ipaddress.ip_address(u'fc00:1::1') == mgmt_intf[('eth0', 'FC00:1::32/64')]['gwaddr'])
+
+    def test_mgmt_device_disable_counters(self):
```

Comment:
Please also check if PORT, ACL, RIF enabled or not exist in config db

Reply:
Thanks, update the testcase.